src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private String fruitName;
+    private int quantity;
+    private Operation operation;
+
+    public FruitTransaction(String fruitName, int quantity, Operation operation) {
+        this.fruitName = fruitName;
+        this.quantity = quantity;
+        this.operation = operation;
+    }
+
+    public String getFruitName() {
+        return fruitName;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public enum Operation {

```

Comment:
The 'Operation' enum is a part of 'FruitTransaction' class, which is a model. It's good practice to separate different types of components into different files. Consider moving 'Operation' enum to its own file for better separation of concerns and easier maintainability.